### PR TITLE
Add HF Spaces demo post

### PR DESCRIPTION
**Pull request by monatis**
Ready for review.

## Notes
- `draft` is set to `false`, so merging this PR will publish it.
- When publishing, we need to make [the accompanying repo](https://github.com/qdrant/hf-spaces-demo) public.

### CODE REVIEW

**Comment by generall** [comment]
Images are not displaying anywhere: 
- empty image on the blog post cover
- empty images on the HF search result

**Comment by monatis** [comment]
Looking into it.

**Comment by monatis** [comment]
Found the root causes of both, fixing.

**Comment by monatis** [comment]
Fixed both. Ready for review.

**Review by kacperlukawski — CHANGES_REQUESTED** [review]
Looks fine to me, I made some minor comments only. I'd like to know about the demo layout. Right now, search results are displayed in the right column, and that requires scrolling the page down over and over. Can we maybe make the search options centred and display the results under the search bar? So the images are displayed in 3-4 columns, and the page is just shorter.

File: qdrant-landing/content/articles/hf-spaces-demo.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+title: Faster Web Demos with Qdrant Cloud and HF Spaces
+short_description: "Building a web demo for your similarity search models in no time"
+description: Learn how to build a public web application for your vector search solution by using Qdrant Cloud and Huggingface Spaces quickly and easily.
+social_preview_image: /articles_data/hf-spaces-demo/social_preview.jpg
+small_preview_image: /articles_data/hf-spaces-demo/icon.svg
+preview_dir: /articles_data/hf-spaces-demo/preview
+weight: 10
```
[the remark this answers is not in the document]
Comment (monatis) [reply qdrant-landing/content/articles/hf-spaces-demo.md]: I Set it to 1, as the most recent was 2.

File: qdrant-landing/content/articles/hf-spaces-demo.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+app on HF Spaces for semantic image search,
+backed by Qdrant Cloud. If you would like to prefer
+the source code directly instead, go to the [project repository](https://github.com/qdrant/hf-spaces-demo).
+
+## Step 1: Setting up
+
+Before starting, make sure that you signed up at Qdrant Cloud,
+created a cluster and obtained the host URL and API key.
+We will use them for accessing our instance, so let's set them as environment variables.
+Do not include the protocol prefix `https://` in the host value, e.g., `{cluster_id}.us-east-1-0.aws.cloud.qdrant.io`.
```
[the remark this answers is not in the document]
Comment (monatis) [reply qdrant-landing/content/articles/hf-spaces-demo.md]: Fixed.

**Pull request closed without merging**